src/style.css

```diff
@@ -0,0 +1,98 @@
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  background: #f4f4f4;
+  font-family: 'Open Sans', 'Lucida Grande', sans-serif;
+}
+
+.main-parent {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  height: 100vh;
+  margin-top: 32px;
+}
+
+h1 {
+  text-align: center;
+}
+
+.to-do-container {
+  width: 400px;
+  background-color: white;
+  box-shadow: 0 5px 8px 0 #c9c9c930, 0 6px 20px 0 #c9c9c930;
+}
+
+.task-container {
+  list-style: none;
+  margin-block: 0;
+  padding-inline: 0;
+  font-size: 1rem;
+}
+
+li {
+  border-bottom: 1px solid #dedede;
+  padding: 1rem;
+}
+
+img {
```

Comment:
I'd recommend you make the image appear like a  move on hover. This is to ensure you follow the template requirement, You could add the :hover CSS pseudoclass and then set the cursor to move.

Reply:
Thanks for the review 👍 💯